Git-Pull-vs-Git-Fetch.md

```diff
@@ -0,0 +1,18 @@
+<h1>Git Pull vs Git Fetch:</h1>
+
+These two commands are regularly used by git users. Let's see the difference between both commands.
+
+
+A clone is simply a copy of a repository. It is something like where you can download source code from some other repository.
+<b>git fetch</b> is the command that says "bring my local copy of the remote repository up to date."
```

Comment:
I would be a bit more accurate here, because it does not really bring the local copy up-to-date. It only retrieves the latest information _about_ the remote repository.

How about something like this:
"Gets the latest meta-information, such as branches and commits, about the remote repository, without downloading the actual data."